src/platform/utilities/environment/stagingDomains.js

```diff
@@ -19,10 +20,6 @@ const domainReplacements = [
   { from: 'https://www\\.myhealth\\.va\\.gov', to: mhvBaseUrl() },
 ];
 
-// This doesn't include preview because we want to redirect to staging urls
-// in preview
-const prodEnvironments = new Set(['production']);
-
```

Comment:
This was still set to a Vets.gov environment, so I'm not sure what affect it has updating it to function correctly. I appreciate any advise on this one.

Reply:
This should be fine now. We were trying to skip domain replacement in vets.gov, but do it in preview.va.gov. We don't need to do any replacement in production now.